libs/javascript.json

```diff
@@ -67,6 +67,7 @@
   {"id":"js.riot","imports":["riot"],"name":"riot","repo":"riot/riot","tags":["ui-framework"],"tech":["javascript-web"],"status":"model-ready"},
   {"id":"js.rx-lite","imports":["rx"],"name":"rx.lite","repo":"Reactive-Extensions/RxJS","tags":["reactive"],"tech":["javascript-web"],"status":"model-ready"},
   {"id":"js.socketio","imports":["socketio"],"name":"socketio","repo":"socketio/socket.io","tags":["networking"],"tech":["javascript-web"],"status":"model-ready"},
+  {"id":"js.svelte","imports":["svelte"],"name":"svelte","repo":"sveltejs/svelte","tags":["ui-framework"],"tech":["javascript-web"],"status":"model-ready"},
```

Comment:
please change model-ready to awaiting-model

Reply:
Ah, sorry for that.